src/scripts/main.js

```diff
@@ -1,3 +1,335 @@
 'use strict';
 
-// write code here
+function convertSalary(salary) {
+  return +salary.slice(1).split(',').join('');
+};
+
+function convertNumberToSalary(number) {
+  const symbol = '$';
+  const digit = new Intl.NumberFormat('en-US').format(number);
+
+  return symbol + digit;
+}
+
+function sortTableString(columnNumber, inputOrder,
+  inputLastSorted, columnName) {
```

Comment:
```suggestion
function sortTableString(
  columnNumber, 
  inputOrder,
  inputLastSorted, 
  columnName
) {
```

Reply:
It's better to place each argument on a new line if you can't place in a single line.